package.json

```diff
@@ -6,15 +6,26 @@
   "scripts": {
     "build": "rollup --output.format=umd --output.file=css-doodle.js -- src/index.js",
     "minify": "uglifyjs --compress --mangle --output=css-doodle.min.js --ecma 6 -- css-doodle.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "karma start",
+    "test": "karma start --single-run"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/yuanchuan/css-doodle.git"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.4",
+    "gulp": "^3.9.1",
```

Comment:
It seems the gulp package hasn't been used?

Reply:
Oops, forgot to remove it after some experimenting.

Removed.